R/init.R

```diff
@@ -159,10 +163,15 @@ init <- function(data,
   # argument transformations
   ## `modules` - landing module
   landing <- extract_module(modules, "teal_module_landing")
-  landing_module <- NULL
   if (length(landing) == 1L) {
-    landing_module <- landing[[1L]]
+    landing_popup <- landing[[1L]]
```

Comment:
```suggestion
    landing_popup <- landing_popup
```

Reply:
Or remove this line complitely. Previously landing_popup was extracted from modules, now it is a parameter